packages/@n8n/nodes-langchain/nodes/chains/ChainLLM/methods/processItem.ts

```diff
@@ -16,9 +16,10 @@ async function getChatModel(ctx: IExecuteFunctions, index: number = 0): Promise<
 		if (connectedModels.length <= index) {
 			throw new NodeOperationError(
 				ctx.getNode(),
-				`Chat Model not found at index ${index}. Available models: ${connectedModels.length}`,
+				'Please connect a model to the Fallback Model input or disable the fallback option',
```

Comment:
This error message is great. Thanks for updating.
Though we are using this same function to get [the main model](https://github.com/n8n-io/n8n/pull/16617/files#diff-50317944dc44a3f630742c7454c28d023b6a672fc97e40cc0e7c600c615f0890R90) as well, so this error does not make sense in that context.

Reply:
Yes, user would get a different error. But it would clearer if the error matched the expectations of what this function is supposed to do. Especially if this would eventually get reused elsewhere in the code.